keystore/src/entities/mod.rs

```diff
@@ -158,8 +153,19 @@ pub trait EntityBase: Send + Sized + Clone + PartialEq + Eq + std::fmt::Debug {
 
         Ok(ret)
     }
+    // FIXME: the unimplemented panics are intentional for now to help identifying where it needs
+    // to be adjusted to use the transaction instead
+    async fn save(&self, _: &mut Self::ConnectionType) -> CryptoKeystoreResult<()> {
+        unimplemented!("This is implemented just for proteus as of now")
+    }
+    async fn insert(&self, _: &mut Self::ConnectionType) -> CryptoKeystoreResult<Self::AutoGeneratedFields> {
+        unimplemented!("This is implemented just for proteus as of now")
+    }
+    async fn delete(_: &mut Self::ConnectionType, _: StringEntityId<'_>) -> CryptoKeystoreResult<()> {
```

Comment:
The delete originally took an array of ids to delete. We delete a single item in every usage of this function, so it was refactored to take a single id.